### DISCUSSION TITLE
Do we need to set `DecimalContext` when using this library?

### DISCUSSION THREAD

**Discussion by J1ng06** [aarondl/sqlboiler#1121] [Q&A] [answered]
Hi folks, I recently started using `sqlboiler` and I realized that there is a global `DecimalContext` in `sqlboiler` and it's not set to any one of these: https://github.com/ericlagergren/decimal/blob/aca2edc11f73e28a0e93d592cc6c3de4a352a81c/context.go#L120-L176

Is it a problem if I leave the `DecimalContext` unset?

**Comment by stephenafamo** [comment, marked as answer]
I believe the unset context is fine. Afterall, when using the package's `decimal.New()` function, it creates a new decimal with an unset context.
